helix-term/src/keymap/default.rs

```diff
@@ -360,6 +360,17 @@ pub fn default() -> HashMap<Mode, Keymap> {
 
         "C-x" => completion,
         "C-r" => insert_register,
+
+        "up" => move_line_up,
+        "down" => move_line_down,
+        "left" => move_char_left,
+        "right" => move_char_right,
+        "C-left" => move_prev_word_end,
+        "C-right" => move_next_word_start,
```

Comment:
I don't think it's right to restore `C-` mappings but not the `A-` mappings.

Reply:
Though that seems to be the nvim default so maybe it's ok

Reply:
I only added back `C-` keys because like other editors and browsers, `C-` is used to move between word, not `A-` so the familiarity comes from there.

Reply:
Let's remove `C-left`, `C-right` and `pageup`, `pagedown` to match kakoune.